libs/vre/shared/app-analytics/src/lib/datadog-rum/datadog-rum.service.ts

```diff
@@ -54,30 +59,36 @@ export class DatadogRumService {
                 });
 
                 // if session is valid: setActiveUser
+                // FIXME: so that updates to the session state are reflected in the specified user
```

Comment:
is this a fixme for this PR?

Reply:
no, for future PRs. I've created issues on Linear.